src/modules/imageresizer/ui/Models/ResizeOperation.cs

```diff
@@ -105,9 +131,9 @@ public void Execute()
                     encoder.Frames.Add(
                         BitmapFrame.Create(
                             Transform(originalFrame),
-                            thumbnail: null,
+                            originalFrame.Thumbnail,
                             metadata,
-                            colorContexts: null));
+                            originalFrame.ColorContexts));
```

Comment:
If I'm not wrong, only these 2 lines change also fixes the issue. can someone confirm? If yes, we don't need to modify metadata

Reply:
Lines 108 and 110 originally

Reply:
You are right, this fixes the initial issue described in #9885. I refer to this as "Bug category 2" in the PR description.

The rest of the code is required to fix the "Bug category 1". Please find a few sample images here:
[#9885 (comment)](https://github.com/microsoft/PowerToys/issues/9885#issuecomment-958967927)
[#9885 (comment)](https://github.com/microsoft/PowerToys/issues/9885#issuecomment-891829849)
[#14266 (comment)](https://github.com/microsoft/PowerToys/issues/14266#issue-1044400800)

Reply:
Btw: thank you for taking the time to review my PR!